tests/_pyMylib/__init__.py

```diff
@@ -0,0 +1,31 @@
+import ctypes
+from pathlib import Path
+
+from threadpoolctl import LibController, register
+
+
+path = Path(__file__).parent / "my_threaded_lib.so"
+ctypes.CDLL(path)
+
+
+class MyThreadedLibController(LibController):
+    user_api = "my_threaded_lib"
+    internal_api = "my_threaded_lib"
+    filename_prefixes = ("my_threaded_lib",)
+
+    def get_num_threads(self):
+        return getattr(self.dynlib, "mylib_get_num_threads")()
+
+    def set_num_threads(self, num_threads):
+        getattr(self.dynlib, "mylib_set_num_threads")(num_threads)
+
+    def get_version(self):
+        get_version = getattr(self.dynlib, "mylib_get_version")
```

Comment:
```suggestion
        # This function returns the version of the linked library
        # if It is exposed. It will be returned in `threadpool_info`.
        get_version = getattr(self.dynlib, "mylib_get_version")
```